students/BrianNisonger/Lesson3/assignment/test_customer_db.py

```diff
@@ -0,0 +1,169 @@
+    second_customer["name"] = "Bob"
+    second_customer["email_address"] = 'bob@hotmail.com'
+    add_customer(**second_customer)
+    return second_customer
+
+
+def add_third_customer():
+    third_customer = dict(ok_customer)
+    third_customer["customer_id"] = "M3233bb"
+    third_customer["name"] = "Jackson"
+    third_customer["status"] = False
+    add_customer(**third_customer)
+    return third_customer
+
+
+def add_fourth_customer():
+    fourth_customer = dict(ok_customer)
+    fourth_customer["customer_id"] = "M3233cc"
+    fourth_customer["name"] = "Michael"
+    fourth_customer["status"] = False
+    add_customer(**fourth_customer)
+    return fourth_customer
+
+
+def clean_up(customer_id):
+    test_customer = Customer.get(Customer.customer_id == customer_id)
+    test_customer.delete_instance()
+
+
+def test_add_ok_customer():
+    add_customer(**ok_customer)
+    test_customer = Customer.get(
+        Customer.customer_id == ok_customer['customer_id'])
+    assert test_customer.email_address == ok_customer['email_address']
+    assert test_customer.customer_id == ok_customer['customer_id']
+    assert test_customer.first_name == ok_customer['name']
+    assert test_customer.last_name == ok_customer['last_name']
+    assert test_customer.phone_number == ok_customer['phone_number']
+    assert test_customer.home_address == ok_customer['home_address']
+    assert test_customer.status == ok_customer['status']
+    assert test_customer.credit_limit == float(ok_customer["credit_limit"])
+    clean_up(ok_customer['customer_id'])
+
+
+def test_credit_limit_float():
+    bad_customer = dict(ok_customer)
+    bad_customer['credit_limit'] = '$40'
+
+    with pytest.raises(ValueError):
+        add_customer(**bad_customer)
+
+
+def test_add_multiple_customers():
+    add_customer(**ok_customer)
+    second_customer = add_second_customer()
+    test_customer = Customer.get(
+        Customer.customer_id == ok_customer['customer_id'])
+    test_customer1 = Customer.get(
+        Customer.customer_id == second_customer["customer_id"])
+    assert test_customer != test_customer1
+    assert test_customer.first_name == 'Suzie'
+    assert test_customer1.first_name == 'Bob'
+    clean_up(ok_customer["customer_id"])
+    clean_up(second_customer["customer_id"])
+
+
+def test_search_customer():
+    add_customer(**ok_customer)
+    second_customer = add_second_customer()
+    customer_dict = search_customer("W3434ff")
+    second_customer["credit_limit"] = float(second_customer["credit_limit"])
+    assert customer_dict == second_customer
+    clean_up(ok_customer["customer_id"])
+    clean_up(second_customer["customer_id"])
+
+
+def test_search_customer_null():
+    add_customer(**ok_customer)
+    customer_dict = search_customer("12345")
+    assert customer_dict == {}
+    customer_dict = search_customer(7)
+    assert customer_dict == {}
+    customer_dict = search_customer()
+    assert customer_dict == {}
```

Comment:
This would be a good candidate for parameterizing. Pytest also has a parameterize module. Anytime you have more than one assert in a test, you should ask yourself if there is a better way. If one fails, the rest are not run, so you do not get that additional information.